### ISSUE TITLE
[OCTO-20] Add job scheduling

### ISSUE THREAD

**Issue by sadikovi** [sadikovi/octohaven#20]
Job scheduling for multiple runs, e.g. every hour, every day and so on.

**Comment by sadikovi** [comment]
We will add new manager to handle scheduling, new tab in UI for looking at statistics of schedules and canceling. Micro architecture of the feature:

Once job is flagged as being run over a particular schedule, we create runner and set timer for that particular timetable. Runner has a link to the set of jobs that have been scheduled. In `Redis` it will look like this:
```
Runner:
- id
- start time (creation of runner, job is added in the set of runner)
- interval (interval for running periodic jobs)
- delay time (time to create job and schedule it at that time, 
so you will have time to close it, if necessary, e.g. 1-5 min)
- number of jobs failed to schedule
- total number of jobs
- is active
```
Each runner keeps a set of finished jobs.

**Comment by sadikovi** [comment]
We will also need to add option to a creation process, whether we need to create periodic runner or not. I do not think we should change anything in `Job` or `SparkJob` classes.

**Comment by sadikovi** [comment]
Our first approach to scheduling was too difficult and immature. I think we should go for using 
Cron scheduling syntax, and parse dates in Python wrapper. This will also be exposed in UI, with 
some examples and samples of schedules.

**Comment by sadikovi** [comment]
PR for this issue: #37

**Comment by sadikovi** [comment]
fixed in #37
